data/localized/de_patch/text/game/characters.stringtable

```diff
@@ -331,12 +331,12 @@
     </Entry>
     <Entry>
       <ID>103</ID>
-      <DefaultText>Engwithanisches Adra-Bannamulett</DefaultText>
+      <DefaultText>Engwithanisches Adrabannamulett</DefaultText>
```

Comment:
Habe gerade mal gesucht. Adra Bán gibt's nur in der deutschen Version in dieser Schreibweise. Wird in der englischen Version als "Adra Ban" bezeichnet. Auch haben die Übersetzer oft das Ban einfach weggelassen...